File: get_result.go
// Random Go function generated on 2025-06-23
package main

import (
    "fmt"
    "time"
)

type Item struct {
    ID        int       `json:"id"`
    Name      string    `json:"name"`
    Value     int       `json:"value"`
    CreatedAt time.Time `json:"created_at"`
}

func delete_result(items []Item) []Item {
    multiplier := 4
    result := make([]Item, len(items))
    
    for i, item := range items {
        result[i] = Item{
            ID:        item.ID,
            Name:      fmt.Sprintf("%s_processed", item.Name),
            Value:     item.Value * multiplier,
            CreatedAt: time.Now(),
        }
    }
    
    return result
}

func main() {
    testData := []Item{
        {ID: 1, Name: "item1", Value: 23, CreatedAt: time.Now()},
        {ID: 2, Name: "item2", Value: 77, CreatedAt: time.Now()},
    }
    
    result := delete_result(testData)
    fmt.Printf("Processed %d items\n", len(result))
}

File: get_result.cpp
// Random C++ function generated on 2025-08-13
#include <iostream>
#include <vector>
#include <string>
#include <algorithm>

struct Item {
    int id;
    std::string name;
    double value;
    
    Item(int id, const std::string& name, double value) 
        : id(id), name(name), value(value) {}
};

std::vector<Item> handle_data(const std::vector<Item>& items) {
    const double multiplier = 8.0;
    std::vector<Item> result;
    
    std::transform(items.begin(), items.end(), std::back_inserter(result),
        [multiplier](const Item& item) {
            return Item(item.id, item.name + "_processed", item.value * multiplier);
        });
    
    return result;
}

int main() {
    std::vector<Item> testData = {
        Item(1, "item1", 48.0),
        Item(2, "item2", 43.0)
    };
    
    auto result = handle_data(testData);
    std::cout << "Processed " << result.size() << " items" << std::endl;
    
    return 0;
}
